code/hud/hudets.cpp

```diff
@@ -276,20 +282,21 @@ void ai_manage_ets(object* obj)
 void set_default_recharge_rates(object* obj)
 {
 	int ship_properties;
-
+	
 	ship* ship_p = &Ships[obj->instance];
 	ship_info* ship_info_p = &Ship_info[ship_p->ship_info_index];
-
+	mprintf(("settig up ETS for ship %s ...\n", ship_p->ship_name));
```

Comment:
Is this statement going to spam the debug log?

Reply:
Indeed, thought I had removed that but forgot to push it!